src/components/addBooks.js

```diff
@@ -52,11 +52,9 @@ function AddBooks() {
   //   }
   // };
   return (
-    <div>
+    <div className="form">
       <h3>Add new book</h3>
-      <form>
-        <label>Title:</label>
-        <br />
+      <form className="form-input">
         <input type="text" className="title" name="title" onChange={(e) => setTitle(e.target.value)} placeholder="Title" />
         <br />
         <select value={Categories} name="categories" className="categories" onChange={(e) => setAuthor(e.target.value)}>
```

Comment:
- [ ] When I tried to select a category, the selection input didn't highlight the option, it has only a fixed value of `Category`. Kindly investigate this issue.
- [ ] Kindly adjust the styling of this section(colors, background color, and input text) to match the Zeplin design(When I write any text in the input it's the same color as the placeholder).